### PR TITLE
Allow rust_test and rust_binary to depend on cc_library targets.

### PR DESCRIPTION
**Pull request by davidzchen**

### CODE REVIEW

**Comment by davidzchen** [comment]
Test added in bazelbuild/rules_rust#33

**Comment by damienmg** [comment]
I don't really understand what that PR is doing, maybe also add @mhlopko for C++

**Comment by davidzchen** [comment]
Thanks, Damien! :)

File: rust/rust.bzl [hunk 1]
```diff
@@ -366,7 +367,7 @@ def _rust_binary_impl(ctx):
   depinfo = _setup_deps(ctx.attr.deps,
                         ctx.label.name,
                         output_dir,
-                        is_library=False)
+                        allow_cc_deps=False)
```
Comment (asayers) [inline rust/rust.bzl]: Looks like you forgot to actually allow rust_binary rules to depend on cc_library

**Pull request merged**